### PR TITLE
=remote Make use of FlushConsolidationHandler to improve throughput.

### PR DESCRIPTION
**Pull request by He-Pin**
refs: https://github.com/apache/incubator-pekko/issues/645
was: https://github.com/apache/incubator-pekko/pull/650

### CODE REVIEW

**Comment by He-Pin** [comment]
Ran several MultiJvm/test , it passed.

File: remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala [hunk 1]
```diff
@@ -368,6 +369,8 @@ class NettyTransport(val settings: NettyTransportSettings, val system: ExtendedA
 
   private def newPipeline(channel: Channel): ChannelPipeline = {
     val pipeline = channel.pipeline()
+    pipeline.addLast("FlushConsolidationHandler",
+      new FlushConsolidationHandler(FlushConsolidationHandler.DEFAULT_EXPLICIT_FLUSH_AFTER_FLUSHES, true))
```
Comment (pjfanning) [inline remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala:373]: should this be `addFirst`? 

From Netty javadoc:

The [FlushConsolidationHandler](https://netty.io/4.1/api/io/netty/handler/flush/FlushConsolidationHandler.html) should be put as first [ChannelHandler](https://netty.io/4.1/api/io/netty/channel/ChannelHandler.html) in the [ChannelPipeline](https://netty.io/4.1/api/io/netty/channel/ChannelPipeline.html) to have the best effect.

Comment (He-Pin) [reply remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala:373]: The others is using addLast too, so it's the effective first. But I think I can change it to add first for more clearly ?

Comment (He-Pin) [reply remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala:373]: @pjfanning I have change the code to make use of `addFirst`

**Review by pjfanning — APPROVED** [review]
lgtm

**Pull request merged**